typescript/infra/config/environments/mainnet3/balances/desiredRelayerBalanceOverrides.json

```diff
@@ -1,8 +1,12 @@
 {
```

Comment:
there's a process for removing these right? are any of the existing ones eligible for getting removed?

Reply:
I'm not totally sure what the process is exactly, and I would've thought it would be ad-hoc inbetween deploys and not as part of a new deploy itself

Reply:
ticked to do this afterwards https://linear.app/hyperlane-xyz/issue/BACK-122/review-desired-relayed-balance-overrides-and-remove-any-where-possible